contrib/index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Have fun ♥️</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+
+    <header>
+      <h1>Hey there! Thanks for contributing to <a href="/">Bitters</a></h1>
+    </header>
+
+    <main>
+
+      <div class="wrapper">
+
+        <p>The purpose of this HTML is to help determine what default settings are with Bitters and to make sure that all possible HTML Elements are included in this HTML so as to not miss any possible Elements when designing a site.</p>
```

Comment:
Also: I think our goal here is to include elements that Bitters _affects_, rather than _all_ HTML elements (in which, I think we are missing quite a few).